release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -419,6 +419,12 @@ For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features
 [id="ocp-release-notes-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4.18-installation-and-update-byoip-ovn_{context}"]
+==== Bring your own IPv4 (BYOIP) join subnet feature is enabled on `ovn-kubernetes` for Virtual Private Cloud (VPC)
+With this release, you can configure the IPv4 join subnet that is used internally by `ovn-kubernetes` when installing a cluster into an existing Virtual Private Cloud (VPC). You can set the `internalJoinSubnet` parameter in the `install-config.yaml` file and deploy the cluster into an existing VPC.
```

Comment:
This feature is not limited to BYO-VPC:
```suggestion
With this release, you can configure the IPv4 join subnet that is used internally by `ovn-kubernetes` when installing a cluster. You can set the `internalJoinSubnet` parameter in the `install-config.yaml` file and deploy the cluster into an existing VPC.
```

Reply:
+1 it is not limited to byo-vpc. That said, byo vpc is the primary use case. Suggested changes look good.